CHANGELOG

```diff
@@ -1,3 +1,13 @@
+snowplow-unified 0.2.0 (2024-01-03)
```

Comment:
Isn't this supposed to be the patch release and the other PRs going into 0.2.0?

Reply:
Right, it should, I changed it to 0.1.3 now. I thought the screen engagement feature should go in a minor release (0.2) but Ryan said that it can go in 0.1.3, so this should go along with it.